routes/blog.js

```diff
@@ -1,55 +1,13 @@
-const articleModel = require('../models/blog')
 const express = require('express')
 const app = module.exports = express()
-const bodyParser = require('body-parser')
+const blog = require('../controllers/blog')
 
+app.get('/blog', blog.getAllArticles)
 
-// Read for the blog CRUD operation 
-//on the articles page list all article
-app.get('/blog', (req, res) => {
-  articleModel.find()
-    .then((result) => {
-      res.send(result)
-    })
-    .catch((err) => {
-      console.log(err)
-    })
-})
+app.get('/blog/:articleId', blog.getSingleArticle)
 
-//Delete  for the CRUD operation
-//delete the article by it's id
-app.delete('/blog/delete/:articleId', (req, res) => {
-  articleModel.findByIdAndDelete(req.params.articleId)
-    .then((result) => {
-      if (!result) {
-        res.send(`article with id ${req.params.articleId} can't be found`)
-      }
-      else {
-        res.send('article deleted successfully')
```

Comment:
Try to format your response as an object, instead of it being a string. This applies to all your responses.
for example, 

res.send({
            status : "success",
            code: 200,
            message : "Article deleted"
        })

Reply:
will work on that for sure!